src/specs/models/processing-config-bodies/ProcessingConfigClassifier.v1.yaml

```diff
@@ -11,5 +11,15 @@ properties:
         type: number
       bandwidth:
         type: number
+additionalProperties:
+  type: object
+  properties:
+    filterSettings:
+      type: object
+      properties:
+        minProbabiliy:
+          type: number
+        bandwidth:
+          type: number
```

Comment:
Why was this factored into a separate object? Also, can you give me an example object that matches this schema?